scripts/tierchecks.js

```diff
@@ -122,6 +122,33 @@ tier_checker.add_new_check(INCLUDING, ["BW2 NU"], function evioliteCheck(src, te
     }
 });
 
+tier_checker.add_new_check(INCLUDING, ["Red/Blue", "Yellow", "Stadium", "RBY Ubers", "RBY OU", "RBY BL", "RBY UU", "RBY NU", "RBY LC"], function noTradebacksCheck(src, team, tier) {
+    var ret = [];
+    var illegal_combinations = {"Kakuna": {"Harden": ["Poison Sting", "String Shot"]}, "Beedrill": {"Harden": ["Poison Sting", "String Shot"]},  "Fearow": {"Payday": ["Peck", "Mirror Move", "Drill Peck"]}, "Nidoking": {"Thrash": ["Fury Attack"]}, "Rapidash": {"Payday": ["Tail Whip", "Growl"]}, "Exeggutor": {"Stomp": ["Sleep Powder", "Stun Spore", "Poison Powder"]}, "Jolteon": {"Focus Energy": ["Thundershock"]}, "Flareon": {"Focus Energy": ["Ember"]}};

```

Comment:
If we do end up fixing it via this route, Fearow should only be restricted on Peck, as that is the only 100% confirmed one for it to be locked out of. magic9mushroom brought up a good point regarding this that makes sense (The Fearow is at level 20, not 40, which still allows it to get Drill Peck and Mirror Move)